presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/DockerContainer.java

```diff
@@ -404,8 +400,6 @@ public void tryStop()
         catch (ExecutionException | RuntimeException e) {
             log.warn("Could not stop container correctly: %s", getStackTraceAsString(e));
         }
-
-        checkState(!isRunning(), "Container %s is still running", logicalName);
```

Comment:
why this one is removed?

Reply:
That's not necessary. Container will be killed eventually (either by just executed command, or by the next environment being started (there is a cleanup before that) or by ryuk). This check is redundant.